packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -958,8 +958,11 @@ mixin WidgetInspectorService {
 
     SchedulerBinding.instance.addPersistentFrameCallback(_onFrameStart);
 
-    final FlutterExceptionHandler structuredExceptionHandler = _reportError;
     final FlutterExceptionHandler defaultExceptionHandler = FlutterError.onError;
+    final FlutterExceptionHandler structuredExceptionHandler = (FlutterErrorDetails details) {
```

Comment:
The downside is this will result in displaying errors twice in IDEs (once with structure, once as raw text).
Maybe what we want to do is call the original exception handler if someone else has overridden the default handler?